### ISSUE TITLE
Issues

### ISSUE THREAD

**Issue by acabal** [b-t-k/william-shakspeare_the-merry-wives-of-windsor#1]
- IA source missing from content.opf. Please also include the 1923 source too, since we've merged them.

- `se clean` is creating unexpected output in act 2 because your blockquote, a block element, is in line with flow elements (i.e. text). In this case, you should not separate the speech into multiple `<tr>`s, as that is not semantically correct. Instead, wrap the speech in `<p>` tags, remove the next `<tr>` that is a run-on of the same speech and also wrap that in a `<p>` tag, and then `se clean` should do the right thing.

- Missing CSS for scene settings. Note how when viewed in a browser `Windsor. Before Page’s house.` is left aligned. See The Importance of Being Earnest act 1 for scene CSS.

- Lots of `<span>`s with leading white space. Ex: `<span> I combat challenge`

- The typography manual requires that em dashes have no white space around them, however the `se typogrify` tool skips that in the instances where an em dash is preceded by a period. This is because often when an em dash is preceded by a period, it's a citation, not a pause in speech. In this case that just means that `se typogrify` guessed wrong and a human eye was required. So, you should remove all spaces around all em dashes, and there are quite a few.

- Some `<br>`s in colophon are misaligned

- Add "The" before PG Shakespeare Team in colophon

- Some misaligned `</td>`s in acts 2 and 3. They are on their own line and not indented for some reason.

- In dc:description, can you summarize in one sentence what the actual play is about, instead of including a meta-detail about the character and the publishing history?

- Dramatis Personae is frontmatter, so its semantics have to be recharacterized, a half title has to be added, and the ToC has to be updated.

**Comment by b-t-k** [comment]
- With the PG Shakespeare editions there is no IA source. They are in effect original editions based, I assume, on their own scholarship and the various quartos and folios. I've added the 1923 edition...

- In cases where an em dash preceded a stage direction I think we should leave the following space in. "No Certainly.— [Aside to her.]" vs. "No Certainly.—[Aside to her.]" I took them out but can put them back in if you think it is more correct.

- I added the half title and redid the T0C.

- fixed all the other issues

- Sorry about the sloppiness, with spaces and tds. I had meant to clean that up but I kept getting distracted by weird issues.
